php/commands/media.php

```diff
@@ -299,16 +299,16 @@ private function _process_regeneration( $id, $skip_delete = false, $only_missing
 			}
 
 			if ( empty( $metadata ) ) {
-				WP_CLI::warning( "Couldn't regenerate thumbnails for $att_desc." );
+				WP_CLI::warning( "$progress Couldn't regenerate thumbnails for $att_desc." );
```

Comment:
I removed the default argument value in 01bebd9, so I should never be empty now.